Sources/FirebaseAndroid/include/FirebaseAndroid.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#ifndef SwiftFirebase_FirebaseAndroidShim_FirebaseAndroid_h
+#define SwiftFirebase_FirebaseAndroidShim_FirebaseAndroid_h
+
+#include <jni.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+jobject SwiftFirebase_GetActivity(void);
+JNIEnv *SwiftFirebase_GetJavaEnvironment(void);
```

Comment:
Could we add `_Nonnull` annotations here and below as well if we want to use these methods from Swift?

Reply:
I don't think that we can ... we can do `_Null_unspecified`. They aren't supposed to be null, but _can_ be null if used improperly.

Reply:
fair enough, I think that's fine then. We could also make them `_Nullable` just to be safe too and get Swift to check them.